src/components/DevelopmentGuide/DesignTokens.mdx

```diff
@@ -88,32 +86,33 @@ To pass style attributes to custom components and native HTML elements, the
 React `style` prop must be used rather than the individual Javascript style
 props. The `style` prop can be populated in a number of ways.
 
-```tsx
+<Source
+  code={`
 // Style values assigned to variables
 const bg = "var(--nypl-colors-ui-bg-default)";
 const color = "var(--nypl-colors-ui-typography-heading)";
 const fontWeight = "var(--nypl-fontWeights-medium)";
 const padding = "var(--nypl-space-inset-default)";
-
+// ...
 <div
   style={{
     backgroundColor: bg,
     color: color,
     fontWeight: fontWeight,
     padding: padding,
   }}
-/>;
-
+/>
+/******************/
 // Style values assigned to a data object
 const stylesDirectlyAssigned = {
   backgroundColor: "var(--nypl-colors-ui-bg-default)",
   color: "var(--nypl-colors-ui-typography-heading)",
   fontWeight: "var(--nypl-fontWeights-medium)",
   padding: "var(--nypl-space-inset-default)",
 };
-
-<div style={stylesDirectlyAssigned} />;
-
+// ...
+<div style={stylesDirectlyAssigned} />
+/******************/
```

Comment:
No blank lines in the code examples, right?

Reply:
Sadly. There might be a config to fix this but I have not found any solution.

Reply:
I split it up to it is easier to read.